modules/gdscript/gdscript.cpp

```diff
@@ -50,11 +41,22 @@
 #include "tests/gdscript_test_runner.h"
 #endif
 
+#include "core/config/engine.h"
+#include "core/config/project_settings.h"
+#include "core/core_constants.h"
+#include "core/core_string_names.h"
+#include "core/io/file_access.h"
+#include "core/io/file_access_encrypted.h"
+#include "core/os/os.h"
+
 #ifdef TOOLS_ENABLED
-#include "editor/editor_paths.h"
 #include "editor/gdscript_docgen.h"
+
+#include "editor/editor_paths.h"
```

Comment:
This can be a bit awkward as those two `editor` folders are different:
- local `editor` subfolder in `modules/gdscript/editor`
- "absolute" `editor` folder with non-module code

This pattern can be found in a few places and requires a bit of brain processing to properly separate the local and global includes, but it's not too bad IMO. Can't really think of a better alternative given the style we're trying to apply.